packages/account-v2/src/containers/AccountClosureSteps/AccountClosureSteps.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, Text } from '@deriv-com/ui';
+import { isNavigationFromDerivGO } from '../../utils/platform';
+
+export const AccountClosureSteps = () => {
+    return (
+        <div>
+            <section className='flex flex-col gap-20'>
+                <Text as='p' size='sm'>
+                    If you close your account:
+                </Text>
+                <ul className='list-disc flex flex-col gap-10 pl-22'>
+                    {/* Cannot use `can't` due to presence of apostrophe*/}
+                    <Text as='li' size='sm'>
+                        You can&apos;t trade on Deriv.
+                    </Text>
+                    <Text as='li' size='sm'>
+                        You can&apos;t make transactions.
+                    </Text>
+                </ul>
+                <Text as='p' size='sm'>
+                    Before closing your account:
+                </Text>
+                <ul className='list-disc flex flex-col gap-10 pl-22'>
+                    <Text as='li' size='sm'>
+                        Close all your positions
+                    </Text>
+                    <Text as='li' size='sm'>
+                        Withdraw your funds
+                    </Text>
+                </ul>
+                <Text as='p' size='sm'>
+                    We shall delete your personal information as soon as our legal obligations are met, as mentioned in
+                    the section on Data Retention in our{' '}
+                    <a
+                        className='text-solid-red-0 font-bold'
+                        href='https://deriv.com/tnc/security-and-privacy.pdf'
+                        rel='noopener noreferrer'
+                        target='_blank'
+                    >
+                        Security and privacy policy
+                    </a>
+                </Text>
+            </section>
+            <section className='mt-24 flex gap-x-16 justify-end'>
+                {!isNavigationFromDerivGO() && (
```

Comment:
can we not call function inside components, we can declare const on the top level and call it down here.